frontend/src/components/Markdown.tsx

```diff
@@ -8,34 +8,41 @@ interface MarkdownProps extends ReactMarkdownOptions {
 }
 
 const Markdown: FunctionComponent<MarkdownProps> = (props) => {
+  const eventDetailsBodyClassName = findClassByName('EventDetailsBody') || undefined;
+  const eventLinkClassName = findClass('43088', 'Link');
```

Comment:
`findClass` & `findClassByName` was moved to head of method to avoid unnecessary search for each `a` tag.